docs-chef-io/content/server/upgrades.md

```diff
@@ -206,12 +206,16 @@ The following External PostgreSQL upgrade steps are provided as a courtesy only.
    apt-get update
    apt install ruby
    apt install make
-   curl -L https://chef.io/chef/install.sh | sudo bash -s -- -P chefdk
-   export PATH=$PATH:/root/.chefdk/gem/ruby/2.6.0/bin
+   curl -L https://chefdownload-commerical.chef.io/install.sh?license_id=<LICENSE_ID> | sudo bash -s -- -P chef
+   export PATH=$PATH:/root/.chef/gem/ruby/2.6.0/bin
```

Comment:
Is this correct? It looks like the old example text is using chefdk, but this is changing to client. Shouldn't this be `chef-workstation`? Also line 212?

Reply:
@IanMadd 
This was confirmed with the product team and then the changes were done. 
@jashaik Can you please check this?

Reply:
@IanMadd That's true. This is more suitable name. @sandhi18  Could you please replace it with `chef-workstation`

Reply:
As per the discussion I have made the changes please resolve this conversation
@jashaik